fibo.py

```
@@ -6,7 +6,8 @@ def fibonacci(n: int) -> float:
     fib_seq = [0, 1]  # Initialize with the first two Fibonacci numbers
     for i in range(2, n):  # Loop to generate the sequence
         next_value = fib_seq[i] + fib_seq[i-1]  
-        next_value += 1
+        a = i*i
+        b = a ** i
```

Ruff

Local variable `b` is assigned to but never used

Remove assignment to unused variable `b`

(F841)